plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/ConnectionWrapper.java

```diff
@@ -99,14 +100,14 @@ void initWith(List<IProject> resources) {
 
 	public boolean load(IExceptionHandler handler) {
 		if (state.compareAndSet(LoadingState.INIT, LoadingState.LOADING)) {
-			startLoadJob(handler);
+			startLoadJob(getWrapped(), handler);
 			return true;
 		}
 		return false;
 	}
 
 	void startLoadJob(ProjectWrapper projectWrapper, IExceptionHandler handler) {
-		new Job("Load project contents") {
+		new Job(NLS.bind("Loading resources for project {0}...", projectWrapper.getWrapped().getName())) {

```

Comment:
Given the fact that the info is displayed in a general context (jobs), I would change to:
Loading resources for Openshift project...